dashboard/app/mailers/ops_mailer.rb

```diff
@@ -34,9 +34,9 @@ def workshop_reminder(workshop, recipient)
     @workshop = workshop
     @recipient = recipient
 
-    subject = "Important: Your #{@workshop.phase_info[:long_name]} workshop is coming up in
```

Comment:
I am kind of grumpy about this code too (in the mailer view)
`
- elsif @workshop.phase && (@workshop.phase >= 4 && @workshop.phase <= 8)
`